modules/oe_bootstrap_theme_paragraphs/config/post_updates/00003/field.storage.paragraph.oe_bt_layout.yml

```diff
@@ -0,0 +1,19 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - paragraphs
+id: paragraph.oe_bt_layout
+field_name: oe_bt_layout
```

Comment:
I am not happy with the field name.
In the future we might want to create other fields named `oe_bt_layout` which would not be integer fields, but a select similar to the "variant". To me "layout" does not imply an integer field.
Alternative would be "oe_bt_n_columns".

But: If this was already agreed on with @drishu and @brummbar then I accept it.

Reply:
In fact this field was formerly the 'variant' field. https://github.com/openeuropa/oe_bootstrap_theme/pull/123/commits/41a02e6d538438fdd749bff4d8a746ac963b461e

To check if is worth the time of changing the name again.

Reply:
Conclusion from @brummbar : Yes we use oe_bt_n_columns. And label can be "Number of columns".
(There was an idea to already use oe_w_* prefix in inticipation of the move, but imo it is cleaner to stick with oe_bt_* while the code is here)